pages/collections3/[collectionAddress]/nfts.tsx

```diff
@@ -0,0 +1,99 @@
+import { CollectionRaisedCard } from '@/common/components/collections/CollectionRaisedCard';
```

Comment:
I really like the layouts here. easy to understand and I feel like I can jump in.